rpcHub/rpcHub.go

```diff
@@ -113,6 +113,8 @@ func dialRpcClient(servAddr string, portNo uint16) (*rpc.Client, net.Conn) {
 // Get a client to the rpc server
 func Client(servAddr string, portNo uint16) *RpcClient {
 	clientKey := fmt.Sprintf("%s:%d", servAddr, portNo)
+	dbLock.Lock()
+	defer dbLock.Unlock()
```

Comment:
Yes, figured that. If this is not called frequently, we are good. I was not sure of the calling pattern of Client().